opensquirrel/decomposer/cnot_decomposer.py

```diff
@@ -37,55 +37,39 @@ def decompose(self, g: Gate) -> list[Gate]:
         # This gives us an ABC decomposition (U = AXBXC, ABC = I) of the target gate.
         # See https://threeplusone.com/pubs/on_gates.pdf
 
-        # Try special case first, see
-        # https://arxiv.org/pdf/quant-ph/9503016.pdf lemma 5.5
+        # Try special case first, see https://arxiv.org/pdf/quant-ph/9503016.pdf lemma 5.5
         controlled_rotation_times_x = general_merger.compose_bloch_sphere_rotations(X(target_qubit), g.target_gate)
-        (
-            theta0_with_x,
-            theta1_with_x,
-            theta2_with_x,
-        ) = ZYZDecomposer().get_decomposition_angles(
-            controlled_rotation_times_x.angle, controlled_rotation_times_x.axis
+        theta0_with_x, theta1_with_x, theta2_with_x = ZYZDecomposer().get_decomposition_angles(
+            controlled_rotation_times_x.angle,
+            controlled_rotation_times_x.axis,
         )
         if abs((theta0_with_x - theta2_with_x) % (2 * math.pi)) < ATOL:
             # The decomposition can use a single CNOT according to the lemma.
-            A = [  # noqa: N806
-                Ry(target_qubit, Float(-theta1_with_x / 2)),
-                Rz(target_qubit, Float(-theta2_with_x)),
-            ]
-            B = [  # noqa: N806
-                Rz(target_qubit, Float(theta2_with_x)),
-                Ry(target_qubit, Float(theta1_with_x / 2)),
-            ]
+            gate_a = [Ry(target_qubit, Float(-theta1_with_x / 2)), Rz(target_qubit, Float(-theta2_with_x))]
```

Comment:
These are not gates, but lists of gates that are summarized as a single operation. I would like to ignore PEP8 here, so we can call them by how these linear operators are known in the theory, i.e., referring to "This gives us an ABC decomposition (U = AXBXC, ABC = I) of the target gate.".

Reply:
Fair enough.